aea/cli/run.py

```diff
@@ -124,14 +124,16 @@ def _verify_ledger_apis_access() -> None:
         logger.debug("No fetchai ledger api config specified.")
     else:
         fetchai_ledger_api_config = cast(LedgerAPIConfig, fetchai_ledger_api_config)
-        _try_to_instantiate_fetchai_ledger_api(fetchai_ledger_api_config.addr, fetchai_ledger_api_config.port)
+        _try_to_instantiate_fetchai_ledger_api(cast(str, fetchai_ledger_api_config.args.get('address')),
+                                               cast(int, fetchai_ledger_api_config.args.get('port')))
 
     ethereum_ledger_config = aea_conf.ledger_apis.read(ETHEREUM)
     if ethereum_ledger_config is None:
         logger.debug("No ethereum ledger api config specified.")
     else:
         ethereum_ledger_config = cast(LedgerAPIConfig, ethereum_ledger_config)
-        _try_to_instantiate_ethereum_ledger_api(ethereum_ledger_config.addr, ethereum_ledger_config.port)
+        _try_to_instantiate_ethereum_ledger_api(cast(str, ethereum_ledger_config.args.get('address')),
+                                                cast(int, ethereum_ledger_config.args.get('port')))
```

Comment:
Now that we have args, why do you not name it correctly (i.e. chain_id rather than port)?